packages/logo/src/docs/Description.mdx

```diff
@@ -0,0 +1,48 @@
+import { Container, Row, Col } from 'storybook/blocks';
+import Logo from '../Logo';
+
+## Colors
+
+Use the `color` prop to change the visual color of the Logo.
+You can set the value to `primary`, `secondary` , `tertiary`
+
```

Comment:
add following line here : 
```tsx live

Reply:
you can check an example here : https://raw.githubusercontent.com/hey-car/heycar-uikit/main/packages/button/src/docs/Description.mdx

Reply:
Yes it was live earlier but after conversation with @derek-heycar we decided we didn't need it for the logo